tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs

```diff
@@ -110,6 +110,10 @@ public void WhenRuntimeParametersArePassedToInstallScripts(string runtime, strin
         [InlineData("Current", "aspnetcore")]
         [InlineData("LTS", "aspnetcore")]
         [InlineData("master", "aspnetcore")]
+        [InlineData("2.2", "aspnetcore")]
+        [InlineData("3.0", "aspnetcore")]
+        [InlineData("3.1", "aspnetcore")]
+        [InlineData("5.0", "aspnetcore")]
```

Comment:
Same question here: Are `"release/3.0", "aspnetcore"`, `"release/3.1", "aspnetcore"`, and `"release/5.0", "aspnetcore"` really still broken?

Reply:
They are still broken. The cause is they are not published to azure blob. 
`aspnetcore\Runtime\release` has only 2.1 and 2.2
`Runtime\release` has 1.0.0, 1.1.0, 2.0.0, 2.1, 2.2, 3.0, 3.1 and uwp6.0(?) 

@mmitche is it expected behavior? Are those release/XX channels are not expected anymore or is it a bug?
SDK still have all the versions.

Reply:
Those should work. @rbhanda @wtgodbe

Reply:
we publish to `aspnetcore\runtime\{BuildVersion}` - should we be copying things to folders based on major.minor? Would that be up to us, or would that be part of release staging?

Reply:
Moved conversation to https://github.com/dotnet/install-scripts/issues/124